Rename misleading instance variable names in FeedbackResponseAttributes #5456

YongJieYongJie opened the pull request:
Fixes #5456

chowyb commented on src/main/java/teammates/logic/core/FeedbackSessionsLogic.java:
```diff
@@ -894,24 +894,24 @@ private StringBuilder getFeedbackSessionResultsForQuestionInCsvFormat(
-                                       ? fsrBundle.getFullNameFromRoster(response.giverEmail)
-                                       : response.giverEmail;
+                                       ? fsrBundle.getFullNameFromRoster(response.giver)
+                                       : response.giver;
```
Maybe indent these 2 lines 2 spaces to the right as well? It seems to be the convention now that `?` and `:` are not aligned with `=` for assignment.

chowyb commented on src/test/java/teammates/test/cases/logic/StudentsLogicTest.java:
```diff
@@ -411,13 +411,13 @@ public void testAdjustFeedbackResponseForEnrollments() throws Exception {
-                                            feedbackResponse1InBundle.giverEmail,
-                                            feedbackResponse1InBundle.recipientEmail);
+                                             feedbackResponse1InBundle.giver,
+                                             feedbackResponse1InBundle.recipient);
```
These lines seem to have been shifted to the right by 1 space, probably unintended.

chowyb commented on src/test/java/teammates/test/driver/TestProperties.java:
```diff
@@ -81,6 +84,9 @@
             
+            IS_HEADLESS_FIREFOX = Boolean.parseBoolean(prop.getProperty("test.is_headless_firefox"));
+            XVFB_PORT = prop.getProperty("test.xvfb_port");
+
```
Not sure if these changes are needed

YongJieYongJie replied:
oops, that's my owning testing configuration

chowyb commented on src/test/java/teammates/test/pageobjects/Browser.java:
```diff
@@ -83,8 +84,16 @@ private WebDriver createWebDriver() {
+                firefoxBinary.setEnvironmentProperty("DISPLAY", TestProperties.XVFB_PORT);
+                return new FirefoxDriver(firefoxBinary, profile);
+            }
+            return new FirefoxDriver(profile);
```
Same here, not sure if these changes are needed

chowyb commented on src/test/resources/data/StudentFeedbackSubmissionEditPageActionTest.json:
```diff
@@ -1456,4 +1456,4 @@
     }
-}
\ No newline at end of file
+}
```
Quite a number of these additional newlines at the end of the file, but I don't think it really hurts the PR so it should be okay.

YongJieYongJie replied:
Actually I don't know why this is showing up, I checked the files and there are no new empty lines at the end of the files.

chowyb commented:
Some minor changes requested, as well as the removal of your headless browser experimentation (pretty sure you didn't plan on pushing that :stuck_out_tongue:)

YongJieYongJie commented:
@chowyb Ready for re-review!
Fixed the indentation and removed my custom testing configurations (was just to irritating to have Firefox popping up when I'm trying to do work while tests run)

chowyb commented:
@YongJieYongJie alright, looks good now :+1:

damithc commented:
@kanghj can you have a look too? I'm not sure if there are hidden problems we missed.

kanghj commented:
Looks good to me. @YongJieYongJie, I would like to request for one more possibly related change though, if you don't mind, in `FeedbackResponse.java`, can you remove the TODO on `private String receiver;`, `//TODO: rename to receiverEmail, will require database conversion`?

YongJieYongJie commented:
@kanghj Removing `// TODO` is always good :thumbsup:

YongJieYongJie commented:
@kanghj Then do we need to attach another `// TODO` (or create an issue) to rename `giverEmail` to `giver` in `FeedbackResponse`?

kanghj commented:
Yes, let's make an issue to rename it.

YongJieYongJie commented:
@kanghj Issue created: 
Rename giverEmail to giver in ResponseFeedback #5571

@damithc Ready for final review!
